src/components/atoms/markdown/CodeFence.tsx

```diff
@@ -53,6 +79,16 @@ const CodeFence = (props: React.HTMLProps<HTMLPreElement> = {}) => {
         >
           <Icon path={mdiContentCopy} />
         </CodeFenceCopyButton>
+        {rawContent && <CodeFenceSaveButton
+          onClick={() => {
+            var blob = new Blob([rawContent], { type: "text/plain;charset=utf-8" });
+            saveAs(blob, "hello world.txt");
```

Comment:
Default file name should be something like `snippet.xx`

Reply:
change it